lua/nvim-tree.lua

```diff
@@ -360,7 +360,8 @@ local function setup_autocommands(opts)
   end
 end
 
-local DEFAULT_OPTS = { -- BEGIN_DEFAULT_OPTS
+local DEFAULT_OPTS = {
+  -- BEGIN_DEFAULT_OPTS
```

Comment:
Please don't move these around; the help update is pretty primitive and expects exact formatting ;)

Reply:
Sorry about that! I think there might be an issue with my `stylua`, potentially because I'm on Windows... but its been telling me that **all** the files in the repository aren't formatted correctly even besides the files I've changed. And moving  "-- BEGIN_DEFAULT_OPTS" was stylua as well. I will try to see if it's possible for me to fix stylua on my end somehow.

Reply:
Whoops, this is silly but it was actually my LSP formatter that was messing with the comment. I thought it was stylua because I also formatted `nvim-tree.lua` through stylua, but nope not stylua's fault there. And I found out why stylua was telling me that all the files weren't formatted correctly, which was because my git was configured to use CRLF line endings (it was probably set to auto), I've changed to use LF line endings instead.

Reply:
No worries. We can't even use `-- stylua: ignore start` `end` as that would also break the generator.